openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/build/BuildConfigOperationsImpl.java

```diff
@@ -163,39 +162,10 @@ public Void trigger(WebHookTrigger trigger) {
   }
 
   @Override
-  public BuildConfigResource<BuildConfig, Void, Build> withResourceVersion(String resourceVersion) {
+  public BuildConfigOperationsImpl withResourceVersion(String resourceVersion) {
     return new BuildConfigOperationsImpl(getContext().withResourceVersion(resourceVersion));
   }
 
-  /*
-   * Labels are limited to 63 chars so need to first truncate the build config name (if required), retrieve builds with matching label,
-   * then check the build config name against the builds' build config annotation which have no such length restriction (but
-   * aren't usable for searching). Would be better if referenced build config was available via fields but it currently isn't...
-   */
-  private void deleteBuilds() {
```

Comment:
Why was this method removed? This seems like a change that's not related to this PR…

Reply:
Only because of the other modifications in this class.  It's been unused for a year and a half.